message RegionRender {
  required uint64 timestep = 1;
  required bytes image = 2;
}

message SendMoreWorldViews{
  required bool enable=1;
}
